client/site-profiler/components/basic-metrics/copies.tsx

```diff
@@ -0,0 +1,384 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { ReactNode } from 'react';
+import type { BasicMetricsScored, Metrics, Scores } from 'calypso/data/site-profiler/types';
+import type { I18N } from 'i18n-calypso';
+
+type SubsetScores = Exclude< Scores, 'needs-improvement' >;
+
+export type MetricsCopies = {
+	[ score in SubsetScores ]: {
+		diagnostic: string | ReactNode;
+		solution: string | ReactNode;
+		cta: string | ReactNode;
+		url: string;
+	};
+};
+
+type WpComNonWpCom = 'wpcom' | 'nonWpcom';
+
+type CopiesProps = {
+	title: string;
+} & {
+	[ key in WpComNonWpCom ]: MetricsCopies;
+};
+
+type CopiesReturnValue = {
+	[ k in Metrics ]: CopiesProps;
+};
+
+export type CopiesReturnValueList = [ Metrics, CopiesProps ][];
+
+export function getCopies(
+	basicMetrics: BasicMetricsScored,
+	translate: I18N[ 'translate' ],
+	domain: string
+): CopiesReturnValue {
+	const migrateUrl = `/setup/hosted-site-migration?ref=site-profiler&from=${ domain }`;
+	const supportUrl = localizeUrl( 'https://wordpress.com/support' );
+
+	const cls: CopiesProps = {
+		title: translate( 'Cumulative Layout Shift (CLS)' ),
+		nonWpcom: {
+			good: {
+				diagnostic: translate(
+					'Your site’s CLS is %(value)f, ensuring a stable layout. Excellent job maintaining low shifts!',
+					{ args: { value: basicMetrics?.cls?.value } }
```

Comment:
This is non-blocking, as we're still discussing the copies on P2. When reading the diagnostic copies, these feel very redundant to the metric displayed above:
<img width="1102" alt="image" src="https://github.com/Automattic/wp-calypso/assets/11555574/b6ded74b-a2be-4935-b8d5-48f76687c9da">

I'd suggest removing the argument here and just displaying a positive/negative message about the metric without repeating the value again.

Reply:
Thanks for your suggestion, I agree it seems redundant. I am going to add your comment to the discussion sheet that I plan to use to gather comments